integration/k8s/run_insecure_test.go

```diff
@@ -60,9 +61,19 @@ func TestRunInsecure(t *testing.T) {
 	}
 	RunTestSteps(orasCpCmds, t)
 
+	err := os.Setenv("IG_EXPERIMENTAL", "true")
+	if err != nil {
+		t.Fatalf("setting IG_EXPERIMENTAL: %v\n", err)
+	}
+
 	// TODO: Ideally it should not depend on a real gadget, but we don't have a "test gadget" available yet.
 	// As the image was not signed, we need to set --verify-image=false.
-	cmd := fmt.Sprintf("$KUBECTL_GADGET run --verify-image=false %s:5000/trace_open:%s -n %s -o json --insecure --timeout 2", registryIP, *gadgetTag, ns)
+	cmd := fmt.Sprintf("ig run --verify-image=false %s:5000/trace_open:%s -n %s -o json --insecure --timeout 2", registryIP, *gadgetTag, ns)
+
+	err = os.Setenv("IG_EXPERIMENTAL", "false")
```

Comment:
Setting the env variable to `true` and then `false` here makes no difference. In line 71 only the string is built and not run.

So it is needed that it will back set back (best to its previous value) after running the `cmd`

Reply:
Totally missed that its not the `main` base branch